### DISCUSSION TITLE
"On this page" sidebar styles / animations

### DISCUSSION THREAD

**Discussion by kevinmichaelchen** [fuma-nama/fumadocs#1211] [Q&A] [answered]
![Capture-2024-12-31-154519](https://github.com/user-attachments/assets/8b42031b-0031-49f8-83bd-ddf1bd0f5cb5)

The sidebar / nav tracker has really elegant animations that track your progress as you scroll.

Is there an easy way to enable that?

**Comment by fuma-nama** [comment, marked as answer]
See `style` prop
https://fumadocs.vercel.app/docs/ui/layouts/page#options
